### PR TITLE
fix(icon): Fix issue with resizing SVGs when jQuery included.

### LINKED ISSUE #1679: Unable to Set Size for md-icon using Iconset and CSS Styling (when using JQuery)
I'm trying to use iconsets as described on the angular material demo for [md-icon](https://material.angularjs.org/#/demo/material.components.icon).

I grabbed the demo `core-icons.svg` file.

I included the following code in my angular app:
```javascript
		.config(['$mdIconProvider', function($mdIconProvider) {
			$mdIconProvider
				.iconSet('core', '../styles/images/icons/material-design/core-icons.svg', 24)
		}])
```

My html before styling:
```html
	<md-button ng-click="vm.showNewDialog($event)">
		<md-icon md-svg-src="core:alarm" style="color: yellow"></md-icon>
		<span>New</span>
	</md-button>
```

**Result**:
image

*Now I try to use CSS properties to resize the icon:*

My html:
```html
	<md-button ng-click="vm.showNewDialog($event)">
	         <md-icon md-svg-icon="core:alarm"
		          style="color: yellow; height: 14px; width: 14px"></md-icon>
		<span>New</span>
	</md-button>
```
**Result**:
image

However, my image does not resize, the md-icon element just gets resized, clipping the md-icon.

Using the chrome inspector, we can see this:
image

Oddly, loading icons from svg URLs **does resize the icons** if we apply css styles to them:
```html
	<md-button ng-click="vm.showNewDialog($event)">
		<md-icon md-svg-src="styles/images/icons/material-design/content/ic_add_24px.svg"
				style="color: yellow; width: 10px height: 10px;"></md-icon>
		<span>New</span>
	</md-button>
```
image

Has anybody else experienced this or found a solution?

### PR DESCRIPTION
**Pull request by programmist**
This fixes #1679.  I replaced most usages of jqLite in the service with native alternatives.

### CODE REVIEW

**Comment by programmist** [comment]
I see the failed Firefox tests.  I'll address those and resubmit.  Would be lovely if browsers were consistent, but I guess that train sailed 10+ years ago.

File: src/components/icon/iconService.js [hunk 1]
```diff
@@ -434,27 +427,29 @@
...
+         angular.forEach({
            'fit'   : '',
            'height': '100%',
            'width' : '100%',
            'preserveAspectRatio': 'xMidYMid meet',
-           'viewBox' : svg.attr('viewBox') || ('0 0 ' + iconSize + ' ' + iconSize)
-         })
-         .css( {
+           'viewBox' : this.element.getAttribute('viewBox') || ('0 0 ' + iconSize + ' ' + iconSize)
+         }, function(attr, val) {
```
Comment (gkalpak) [inline src/components/icon/iconService.js]: Switching the argument names would be more intuitive :)

File: src/components/icon/iconService.spec.js [hunk 2]
```diff
@@ -106,22 +106,27 @@ describe('mdIcon service', function() {
...
       'xmlns' : 'http://www.w3.org/2000/svg',
       'fit'   : '',
       'height': '100%',
       'width' : '100%',
       'preserveAspectRatio': 'xMidYMid meet',
-      'viewBox' : svg.attr('viewBox') || '0 0 24 24'
-    })
-    .css( {
+      'viewBox' : svg.getAttribute('viewBox') || '0 0 24 24'
+    }, function(attr, val) {
```
Comment (gkalpak) [inline src/components/icon/iconService.spec.js]: Switching the argument names would be more intuitive :)

Comment (programmist) [inline src/components/icon/iconService.spec.js]: Thanks for catching that.

**Pull request closed without merging**